package.json

```diff
@@ -39,7 +40,28 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-rxjs": ">= 5",
     "eslint-plugin-rxjs-angular": ">= 2",
-    "eslint-plugin-simple-import-sort": ">= 12"
+    "eslint-plugin-simple-import-sort": ">= 12",
+    "typescript": ">= 4"
```

Comment:
Typescript is already included from typescript-eslint, and implicitly depended on. Added it implicitly here.